clusterloader2/pkg/measurement/common/slos/network_programming.go

```diff
@@ -48,6 +48,10 @@ func init() {
 type netProgGatherer struct{}
 
 func (n *netProgGatherer) IsEnabled(config *measurement.MeasurementConfig) bool {
+	//NetworkProgrammingLatency measurement should detect if scraping kube-proxy is disabled and disable itself if so
```

Comment:
// Disable NetworkProgrammingLatency measurement if scraping kube-proxy is disabled.